src/song_utils.py

```diff
@@ -232,3 +232,40 @@ def get_cover_art(filename: str) -> Optional[File]:
 
     # Create a new discord file from the file pointer and name
     return File(image_bytes_fp, filename=cover_filename)
+
+def convert_timestamp_to_seconds(time_str):
+    # Converts a time string into seconds. Returns 0 if format is invalid.
+    # Format is handled either in pure seconds (93, 180) or hh:mm:ss format (1:23:45).
+    if time_str is None:
+        return 0
+    
+    if ':' not in time_str:
+        if not time_str.isdigit():
+            return 0
```

Comment:
imo preferred behavior would be to return `None` on all invalid inputs, and then in the bust command instead of ignoring the parameter, just respond to the interaction refusing to bust bc of malformed input. If someone is using the input, they probably mean to and did not just make a typo and so an error would be better so they can try again, instead of just ignoring their input